raft/src/main/java/org/apache/kafka/raft/QuorumConfigs.java

```diff
@@ -47,7 +47,7 @@
  * controller should be able to transition from standby to active without reloading all of
  * the metadata. The standby is a "hot" standby, not a "cold" one.
  */
-public class RaftConfig {
+public class QuorumConfigs {
```

Comment:
It seems to me QuorumConfig`s` implies that we should not have instance of it. However, we do instantiate it, and so could we rename it to `QuorumConfig`? Another similar case is `LogConfig`.